### ISSUE TITLE
Filter Options for Kleingruppen

### ISSUE THREAD

**Issue by MichaelGSP** [ecclesianuernberg/genesis#19]
Meets on day?
At time?

Near address
Near Zipcode

Age range

Category

Probably we need to enhance CT for that?!

It would be something like this, if possible
https://www.churchofthehighlands.com/groups/directory
(Advanced Search)

**Comment by xsteadfastx** [comment]
an sowas hat doch der jonas gearbeitet...

**Comment by MichaelGSP** [comment]
Ja. Es wird Zeit, dass wir uns zusammen setzen um abzugleichen wer was hat und wer was von wem nutzen kann.
